pkg/volume/aws_ebs/aws_util.go

```diff
@@ -91,12 +92,19 @@ func (util *AWSDiskUtil) CreateVolume(c *awsElasticBlockStoreProvisioner) (aws.K
 	}
 	// Apply Parameters (case-insensitive). We leave validation of
 	// the values to the cloud provider.
+	zonePresent := false
+	zonesPresent := false
+	var calculateZonesParams volume.CalculateSetOfZonesParams
 	for k, v := range c.options.Parameters {
 		switch strings.ToLower(k) {
 		case "type":
 			volumeOptions.VolumeType = v
 		case "zone":
-			volumeOptions.AvailabilityZone = v
+			zonePresent = true
+			calculateZonesParams.StorageClassZones = v
```

Comment:
This will interpret `zone` parameter as comma separated list of zones, right? IMO it's not what we want.